roles/openshift_manage_node/tasks/config.yml

```diff
@@ -9,3 +9,25 @@
   until: node_schedulable is succeeded
   when: "'nodename' in openshift.node"
   delegate_to: "{{ openshift_master_host }}"
+
+- import_role:
```

Comment:
Why is this here?  This should be handled in meta/main.yml.

Reply:
Right, this part is not necessary, removed